opencl/test/unit_test/device/get_device_info_tests.cpp

```diff
@@ -992,3 +993,30 @@ TEST(GetDeviceInfoTest, givenDeviceWithSubDevicesWhenGettingNumberOfComputeUnits
     EXPECT_EQ(expectedComputeUnitsForRootDevice, numComputeUnits);
     EXPECT_EQ(sizeof(numComputeUnits), retSize);
 }
+
+TEST(GetDeviceInfoTest, givenPciBusInfoWhenGettingPciBusInfoForDeviceThenPciBusInfoIsReturned) {
+    PhysicalDevicePciBusInfo pciBusInfo(0, 1, 2, 3);
+
+    auto driverInfo = new DriverInfoMock();
+    driverInfo->setPciBusInfo(pciBusInfo);
+
+    auto device = std::make_unique<MockClDevice>(MockDevice::createWithNewExecutionEnvironment<MockDevice>(nullptr));
+    device->driverInfo.reset(driverInfo);
+    device->initializeCaps();
+
+    cl_device_pci_bus_info devicePciBusInfo;
+
+    size_t sizeReturned = GetInfo::invalidSourceSize;
+    auto retVal = device->getDeviceInfo(CL_DEVICE_PCI_BUS_INFO_INTEL, 0, nullptr, &sizeReturned);
+
+    ASSERT_EQ(retVal, CL_SUCCESS);
+    ASSERT_EQ(sizeReturned, sizeof(devicePciBusInfo));
+
+    retVal = device->getDeviceInfo(CL_DEVICE_PCI_BUS_INFO_INTEL, sizeof(devicePciBusInfo), &devicePciBusInfo, nullptr);
+    ASSERT_EQ(CL_SUCCESS, retVal);
+
+    EXPECT_EQ(devicePciBusInfo.pci_domain, pciBusInfo.pciDomain);
+    EXPECT_EQ(devicePciBusInfo.pci_bus, pciBusInfo.pciBus);
+    EXPECT_EQ(devicePciBusInfo.pci_device, pciBusInfo.pciDevice);
+    EXPECT_EQ(devicePciBusInfo.pci_function, pciBusInfo.pciFunction);
+}
```

Comment:
style: Internal Lint might not allow "no new line"

Reply:
Newline has been added.